File: MPI/mpicheck/tests/functions/2.2/MPI_Op_commutative.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Op var_0;
    int *var_1;
    int ret;
    /* calls */
    ret = MPI_Op_commutative(var_0, var_1);
    ret = PMPI_Op_commutative(var_0, var_1);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.2/MPI_Op_commutative.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       LOGICAL var_1
       INTEGER var_2
        call mpi_op_commutative(var_0, var_1, var_2)
       call pmpi_op_commutative(var_0, var_1, var_2)
        end program main
    